docs/kafka/access-mgmt-kafka/README.adoc

```diff
@@ -511,26 +511,26 @@ h|Operations
 As an organization administrator, you can use Role-Based Access Control (RBAC) to allow users to view other users in an organization.
 
 You set up access by assigning a predefined role called `User Access principal viewer` to a user group.
-By assigning the role, users within the group are able to do the following:
+By assigning the role, users within the group are able to do the following actions:
 
-* View and select other users when changing owners and managing access to Kafka instances in the web console
-* Specify user names when using the `rhoas` CLI for {product-long-kafka}
+* View and select other users when changing owners and managing access to Kafka instances in the {service-url-kafka}[_Openshift Streams for Apache Kafka_^] web console.
+* Specify user names when using the `rhoas` CLI for {product-long-kafka}.
 
 .Prerequisites
-* You're logged into the {org-name} web console as an organization administrator.
+* You're logged into the {cloud-console-url} as an organization administrator.
 * A user group contains the users to assign the role to.
 
 NOTE: If you want to add the `User Access principal viewer` role to a single user, create a new group for that user only.
 
 ifndef::community[]
-For more information on setting up user access in the web console, see the link:https://access.redhat.com/documentation/en-us/red_hat_hybrid_cloud_console/[_User Access Configuration Guide for Role-based Access Control (RBAC)_^].
+For more information on setting up user access in the {cloud-console-url}, see the link:https://access.redhat.com/documentation/en-us/red_hat_hybrid_cloud_console/[_User Access Configuration Guide for Role-based Access Control (RBAC)_^].
```

Comment:
Think we simply want to say 'Red Hat Hybrid Cloud Console' here, instead of using the URL attribute.

Reply:
Per item 5.1 of our [style guidelines](https://docs.google.com/document/d/1TERH4XRH6La44hUQH88LUaax-35JH3dLXmceGJoLv7I/edit#heading=h.v8qjgp8tzewa) we agreed to stop italicizing the names of other Red Hat guides.